src/less/global.less

```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  margin: 0; 
+  padding: 0;
+  font-size: 62.5%;
```

Comment:
Having this applied to the html and the body seems to make the font-size extra small; it could use some up-sizing so that I can see the list items better!